src/SuperMassive/Helpers/SonarHelper.cs

```diff
@@ -0,0 +1,13 @@
+namespace SuperMassive
+{
+    using System;
+
+    public class SonarHelper
+    {
+        // just for the sake of testing sonar quality gate (uncovered)
+        public static void TestMethod()
+        {
+            Console.WriteLine($"{nameof(SonarHelper)}.{nameof(TestMethod)} has been called");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/manual/PulsarBlow/SuperMassive/pullRequest?prid=6193423)